install_config/master_node_configuration.adoc

```diff
@@ -1013,3 +1013,213 @@ To launch a node server using a node configuration file:
 ----
 $ openshift start node --config=/openshift.local.config/node-<node_hostname>/node-config.yaml
 ----
+
+[[config-logging-levels]]
```

Comment:
Since unique IDs must be unique across the whole repo (i.e., all books), you could add something to the front like "master-node-config-" to denote the topic it's part of, in case later some other topic has a similarly titled section. See other IDs in this topic which have a similar prefix.